rust/libnewsboat/tests/configpaths_returns_possible_paths_to_urls_file.rs

```diff
@@ -0,0 +1,39 @@
+extern crate libnewsboat;
+#[macro_use]
+extern crate section_testing;
+extern crate tempfile;
+
+use self::libnewsboat::configpaths::ConfigPaths;
+use std::{env, path};
+
+enable_sections! {
+#[test]
+fn t_configpaths_returns_possible_paths_to_urls_file() {
+    let home = path::PathBuf::from("/absurd/homedir path");
+
+    env::set_var("HOME", &home);
+
+    if section!("XDG_CONFIG_HOME is set") {
+        let config_home = home.join("configurations");
+        env::set_var("XDG_CONFIG_HOME", &config_home);
+
+        let paths = ConfigPaths::new();
+
+        let expected_dotdir = home.join(".newsboat/urls");
```

Comment:
I personally prefer to actually see the expected final absolute paths in the test code rather than building it in my head (and thus scan through several lines). Yes, a failing test would reveal it, too, but it's not the same. ;-) Additionally, the productive code `join(…)`s the paths as well, so there could be a small potential problem since both use the exact same approach.